heclib/heclib_c/src/Internal/zcatSortPath.c

```diff
@@ -119,8 +119,8 @@ int zcatSortPath(int handle, const char *pathname, size_t pathnameLen, int dataT
 			partLen[3] = (int)strlen(cdate);
 		}
 		len = stringCopy (cinterval, sizeof(cinterval), cpart[4], (size_t)partLen[4]);
-		flagDirection = 1;
-		status = ztsGetStandardInterval(7, &interval, cinterval, sizeof(cinterval), &flagDirection);
+		operation = EPART_TO_SECONDS;
```

Comment:
I like that variable name 'operation'

Reply:
I think it was probably originally okay since it was just a flag to say E-part to seconds or seconds to E-part, but the code grew more options and outgrew the parameter name.